examples/data-objects/prosemirror/src/index.ts

```diff
@@ -41,8 +41,6 @@ class ProseMirrorFactory extends RuntimeFactoryHelper {
             existing,
         );
 
-        // Flush mode to manual to batch operations within a turn
-        runtime.setFlushMode(FlushMode.Manual);
```

Comment:
This removal suggest that default mode was changed, yet there is no mention in breaking.md, and I think you mentioned that it's behind feature gate. So should this code stay until it's switched with default settings?

Reply:
Unless we believe it does not matter for this code (which might be the case)

Reply:
I don't believe it matters for these particular examples.